client/app/lib/components/sidebar/index.coffee

```diff
@@ -96,6 +97,9 @@ module.exports = class Sidebar extends React.Component
       when 'Initialize'
         EnvironmentFlux.actions.generateStack(id).then ({ template }) ->
           appManager.tell 'Stackeditor', 'reloadEditor', template._id
+      when 'Clone'
+        new kd.NotificationView { title: 'Cloning Stack Template' }
+        computeController.cloneStackTemplate draft.toJS(), yes
```

Comment:
Let's move this one into an action as well. If we don't, it will be harder to identify these calls when we make the transition from `nuclear` to `redux`